app/ChordNode.ts

```diff
@@ -51,19 +51,19 @@ export class ChordNode {
     this.predecessor = NULL_NODE;
   }
 
-  iAmTheNode(theNode) {
+  iAmTheNode(theNode: Node): boolean {
     return this.id == theNode.id;
   }
 
-  iAmMyOwnSuccessor() {
+  iAmMyOwnSuccessor(): boolean {
     return this.id == this.fingerTable[0].successor.id;
   }
 
-  iAmMyOwnPredecessor() {
+  iAmMyOwnPredecessor(): boolean {
     return this.id == this.predecessor.id;
   }
 
-  encapsulateSelf() {
+  encapsulateSelf(): Node {
```

Comment:
I'll use this in my functions from now on, I didn't notice we had a Node interface.